user/views.py

```diff
@@ -1,3 +1,86 @@
-from django.shortcuts import render
+import jwt
+import bcrypt
+import json
 
-# Create your views here.
+from django.http           import JsonResponse
+from django.views          import View
+from django.db.models      import Q
+
+from user.models           import User
+from my_settings           import SECRET_KEY_JWT, ALGORITHM
+from user.utils            import validate_email, validate_password, validate_phone_number
+from google.oauth2         import id_token
+from google.auth.transport import requests
+
+
+class SocialLoginView(View):
+
+    def post(self, request):
+
+        try:
+            #CLIENT_ID  = request.headers.get('client', None)
+            token        = request.headers.get('token', None)
+            data         = id_token.verify_oauth2_token(token, requests.Request())
+            email        = data['email']
+            user, flag   = User.objects.get_or_create(email=email)
+            if flag:
+                token        = jwt.encode({'id':user.id}, SECRET_KEY_JWT, ALGORITHM)
+                access_token = token.decode('utf-8')
+                context = {
+                    'accessToken': access_token
+                }
+                return JsonResponse({'result':context}, status=200)
```

Comment:
```suggestion
                return JsonResponse({'access_token': token}, status=200)
```
불필요한 nesting을 줄여주는게 프론트 입장에서 좋을 것 같네요.

Reply:
- 이것도 반영해주세요.

Reply:
``` python
return JsonResponse({'accessToken':access_token}, status=200)

카멜케이스로 이번에 작성하기로 약속하여 이렇게 수정하였습니다.